files/create-ecr-repo/src/handler.py

```diff
@@ -100,15 +103,28 @@ def run(event, context):
             sys.exit(1)
 
         try:
-            policy = lifecycle_policy()
-            if policy:
+            lifecycle_policy = get_valid_json_from_env("REPO_LIFECYCLE_POLICY")
+            if lifecycle_policy:
                 client.put_lifecycle_policy(
                     registryId=account_id,
                     repositoryName=repository,
-                    lifecyclePolicyText=policy,
+                    lifecyclePolicyText=lifecycle_policy,
                 )
                 logger.info("created lifecycle_policy on %s", repository)
         except:  # noqa: E722
             logger.error("failed to create lifecycle_policy on %s", repository)
 
+        try:
+            repository_policy = get_valid_json_from_env("REPO_POLICY")
+            if repository_policy:
+                client.set_repository_policy(
+                    registryId=account_id,
+                    repositoryName=repository,
+                    policyText=repository_policy,
+                    force=False,
+                )
+                logger.info("created repository_policy on %s", repository)
+        except:  # noqa: E722
+            logger.error("failed to create repository_policy on %s", repository)
```

Comment:
```suggestion
        except Exception as e:
            logger.error("failed to create repository_policy on %s: %s", repository, e)
```

Reply:
created [`anastas/5-suggestions`](https://github.com/tradeparadigm/terraform-aws-ecr-repo-lambda/tree/anastas/5-suggestions) with all of our nitpicks

Reply:
I suppose it will be good to add the same for lifecycle policy in `except` block :smiley: 

https://github.com/tradeparadigm/terraform-aws-ecr-repo-lambda/blob/main/files/create-ecr-repo/src/handler.py#L111-L112